core/evm.go

```diff
@@ -92,8 +92,8 @@ func GetHashFn(ref *types.Header, chain ChainContext) func(n uint64) common.Hash
 		if hash, ok := cache[n]; ok {
 			return hash
 		}
-		// Not cached, iterate the blocks and cache the hashes
-		for header := chain.GetHeader(ref.ParentHash, ref.Number.Uint64()-1); header != nil; header = chain.GetHeader(header.ParentHash, header.Number.Uint64()-1) {
+		// Not cached, iterate the blocks and cache the hashes (up to a limit of 256)
```

Comment:
Please add a comment here and in `contrac_comm/evm.go` that when one is changed so should the other